crates/client/CHANGELOG.md

```diff
@@ -0,0 +1,275 @@
+# Changelog
```

Comment:
We need frontmatter for the release automation here, if we want to include it in the release. I think we do want to include it, right?